src/Summoner/Template.hs

```diff
@@ -536,15 +536,15 @@ createProjectTemplate ProjectData{..} = Dir (toString repo) $
               export PATH="/opt/ghc/$$GHCVER/bin:/opt/cabal/$$CABALVER/bin:$$PATH"
               echo $$PATH
               cabal new-update
-              cabal new-build --enable-tests --enable-benchmarks
+              cabal new-build --enable-tests --enable-benchmarks --ghc-options=-Werror
             else
               mkdir -p ~/.local/bin
               export PATH="$$HOME/.local/bin:$$PATH"
               travis_retry curl -L 'https://www.stackage.org/stack/linux-x86_64' | tar xz --wildcards --strip-components=1 -C ~/.local/bin '*/stack'
               stack --version
               stack setup --no-terminal --install-cabal 2.0.1.0
               stack ghc -- --version
-              stack build --only-dependencies --no-terminal
+              stack build --only-dependencies --no-terminal --ghc-options=-Werror
```

Comment:
this is `--only-dependencies` so no need to add the option here. You should add it here instead:

https://github.com/kowainik/summoner/blob/6d9cc81078d43fc72c959bfe7659c16952710617/src/Summoner/Template.hs#L552